lib/airbrussh/log_file_formatter.rb

```diff
@@ -0,0 +1,42 @@
+require "delegate"
+require "fileutils"
+require "logger"
+require "sshkit"
+
+module Airbrussh
+  # A Pretty formatter that sends its output to a specified log file path.
+  # LogFileFormatter takes care of creating the file (and its parent
+  # directory) if it does not already exist, opens it for appending, and writes
+  # a delimiter message. The file is automatically rotated if it reaches 20 MB.
+  #
+  class LogFileFormatter < SimpleDelegator
+    attr_reader :path
+
+    def initialize(path, formatter_class=SSHKit::Formatter::Pretty)
+      @path = path
+      ensure_directory_exists
+      super(formatter_class.new(log_file_io))
+      write_delimiter
+    end
+
+    private
+
+    def write_delimiter
+      delimiter = []
+      delimiter << "-" * 75
+      delimiter << "START #{Time.now} cap #{ARGV.join(' ')}"
+      delimiter << "-" * 75
+      delimiter.each do |line|
+        write(SSHKit::LogMessage.new(SSHKit::Logger::INFO, line))
```

Comment:
@robd I believe `SSHKit::Formatter::Abstract#info` is only available in sshkit `master`. https://github.com/capistrano/sshkit/commit/2aa78905f0c521ad9f697e7a4ed04ba438d5ee78#diff-27e602daf30c94b408f408bdc0cc2af4